app/(landing)/content/en.mdx

```diff
@@ -195,6 +195,42 @@
 
 </InDevelopment>
 
+<Testimonials>
+  ## Crypto Investors Love Eonian
+
+  <Testimonials.CTA href="/earn">
+    Save Your Crypto
+  </Testimonials.CTA>
+
+  <Testimonials.Content>
+
+    <Testimonials.Ribbon>
+      <Testimonials.Tweet id="1697597961503490277" noop-comment="Wish to be guided on proper risk management..." />
```

Comment:
"noop-comment" does not do anything, it's just a comment for us (devs). There is no way to use inline comments in MDX, so i decided to put it to noop attribute

Reply:
It's okay, we can left it this way, but for future, comments in mdx is possible to left same way like in jsx code
```suggestion
      <Testimonials.Tweet id="1697597961503490277" /> { /* "Wish to be guided on proper risk management..." */ }
```